app/api/v2/handlers/contact_api.py

```diff
@@ -15,7 +15,20 @@ def add_routes(self, app: web.Application):
         router = app.router
         router.add_get('/contacts/{name}', self.get_contact_report)
 
-    @aiohttp_apispec.docs(tags=['contacts'])
+    @aiohttp_apispec.docs(tags=['contacts'],
+                          summary='Retrieve a List of Beacons made by Agents to the specified Contact',
+                          description='Returns a list of beacons made by agents to the specified contact. The response'
+                                      ' is formatted as a list of dictionaries. The dictionaries have the keys "paw",'
```

Comment:
One small thing here - you can use ` ` (ex. `paw`) to highlight keywords in the docs

Reply:
Added!